src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -144,8 +145,12 @@ export type Fiber = {
 
 };
 
+export type FiberDev = Fiber & {
```

Comment:
Btw, I had to fix this pattern when I did that: https://github.com/facebook/react/pull/8545/files#diff-5842acf7560d3c499ac59f2652b3918dR328